client/src/slate-helpers/slate-utils/isSelectionAcrossPointers.js

```diff
@@ -0,0 +1,16 @@
+import { closestPointerExportAncestor } from "./closestPointerExportAncestor";
+
+export function isSelectionAcrossPointers(selection, document) {
+  const { anchorKey, focusKey } = selection;
+  const anchorNode = document.getNode(anchorKey);
+  const focusNode = document.getNode(focusKey);
+
+  const closestPointerExportAncestorOfAnchorNode = closestPointerExportAncestor(anchorNode, document);
+  const closestPointerExportAncestorOfFocusNode = closestPointerExportAncestor(focusNode, document);
```

Comment:
From some testing, it seems like this returns undefined if this the anchor/focus are not in an export. If this is the case, then ``closest`` seems a bit misleading. Maybe something like, ``containingExportAncestor``?